ArduPlane/make.inc

```diff
@@ -60,3 +60,4 @@ LIBRARIES += AP_Landing
 LIBRARIES += AP_Beacon
 LIBRARIES += PID
 LIBRARIES += AP_Soaring
+LIBRARIES += AP_Volz_Protocol
```

Comment:
you've added this correctly in Plane but forgot the other vehicles

Reply:
it is also not being being initialized or being updated in other vehicles, not sure if relevant for others...

Reply:
that's true. I was asking because Travis was failing while building copter but now it's passing so it's OK.